tests/reorder_python_imports_test.py

```diff
@@ -1135,6 +1135,15 @@ def test_main_stdin_no_fix(capsys):
     assert out == 'import os\nimport sys\n'
 
 
+def test_main_stdin_diff_only(capsys):
+    input_b = b'import os\nimport sys\n'
+    stdin = io.TextIOWrapper(io.BytesIO(input_b), 'UTF-8')
+    with mock.patch.object(sys, 'stdin', stdin):
+        assert main(('-', '--diff-only')) == 0
+    out, err = capsys.readouterr()
+    assert out == ''
```

Comment:
can you add another test demoing when this actually produced a diff?  this test is useful for showing the no-diff case

Reply:
sure